### PR TITLE
nrf_wifi: Edge backoff support using RF params

### PR DESCRIPTION
**Pull request by imapa**
[SHEL-2481]: Support for applying frame format dependent edge backoff in PHY firmware.

[SHEL-2481]: https://nordicsemi.atlassian.net/browse/SHEL-2481?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by srkanordic — APPROVED** [review]

File: nrf_wifi/fw_if/umac_if/inc/fw/patch_info.h [hunk 1]
```diff
@@ -56,7 +56,7 @@ struct nrf70_fw_image_info {
 
 #define RPU_FAMILY         (1)
 #define RPU_MAJOR_VERSION   (2)
-#define RPU_MINOR_VERSION   (9)
-#define RPU_PATCH_VERSION   (25)
+#define RPU_MINOR_VERSION   (10)
+#define RPU_PATCH_VERSION   (0)
```
Comment (sachinthegreen) [inline nrf_wifi/fw_if/umac_if/inc/fw/patch_info.h:60]: SHEL-2512 is not referenced in the commit log for this change.

**Review by sachinthegreen — CHANGES_REQUESTED** [review]

File: nrf_wifi/fw_if/umac_if/src/radio_test/fmac_api.c [hunk 2]
```diff
@@ -134,7 +140,40 @@ enum nrf_wifi_status nrf_wifi_fmac_dev_init_rt(struct nrf_wifi_fmac_dev_ctx *fma
...
+		nrf_wifi_osal_log_err(fmac_dev_ctx->fpriv->opriv,
+				      "%s: Fetching of RPU OTP information failed",
+				      __func__);
+		goto out;
+	}
+
+	nrf_wifi_osal_mem_set(fmac_dev_ctx->fpriv->opriv,
+			             &phy_rf_params,
+			             0xFF,
+			             sizeof(phy_rf_params));
```
Comment (sachinthegreen) [inline nrf_wifi/fw_if/umac_if/src/radio_test/fmac_api.c:162]: Indentation

File: nrf_wifi/fw_if/umac_if/src/radio_test/fmac_api.c [hunk 3]
```diff
@@ -134,7 +140,40 @@ enum nrf_wifi_status nrf_wifi_fmac_dev_init_rt(struct nrf_wifi_fmac_dev_ctx *fma
...
+
+	status = nrf_wifi_fmac_rf_params_get(fmac_dev_ctx,
+						&phy_rf_params);
```
Comment (sachinthegreen) [inline nrf_wifi/fw_if/umac_if/src/radio_test/fmac_api.c:165]: Indentation

File: nrf_wifi/fw_if/umac_if/src/radio_test/fmac_api.c [hunk 4]
```diff
@@ -134,7 +140,40 @@ enum nrf_wifi_status nrf_wifi_fmac_dev_init_rt(struct nrf_wifi_fmac_dev_ctx *fma
...
+
+	if (status != NRF_WIFI_STATUS_SUCCESS) {
+		nrf_wifi_osal_log_err(fmac_dev_ctx->fpriv->opriv,
+					"%s: RF parameters get failed",
+					__func__);
```
Comment (sachinthegreen) [inline nrf_wifi/fw_if/umac_if/src/radio_test/fmac_api.c:170]: Indentation

**Review by sachinthegreen — APPROVED** [review]

**Pull request merged**